NEMO/admin.py

```diff
@@ -111,6 +111,10 @@ class Media:
 		widget=FilteredSelectMultiple(verbose_name="Nonrequired resources", is_stacked=False),
 	)
 
+	_tool_calendar_color = forms.CharField(
+		required=False, max_length=9, initial='#33ad33', label="Tool calendar color",
```

Comment:
The label is not needed here for tool (there is already some custom js that deals with capitalizing)
```suggestion
		required=False, max_length=9, initial='#33ad33',
```